### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

File: tests/test_hv.py [hunk 1]
```diff
@@ -4,28 +4,34 @@
 from unittest.mock import patch
 
 
-def test_arch():
-    hv = virt_lightning.LibvirtHypervisor("test:///default")
+def test_arch(hv):
     assert hv.arch == 'i686'
```
Comment (zloidemon) [inline tests/test_hv.py]: well, it might not work for non linux systems.

File: virt_lightning/__init__.py [hunk 2]
```diff
@@ -66,15 +68,23 @@ def arch(self):
...
+        return sorted(available)[0]
 
     def create_domain(self, name=None, distro=None):
         if not name:
             name = uuid.uuid4().hex[0:10]
         root = ET.fromstring(DOMAIN_XML)
         root.attrib["type"] = self.domain_type
         root.find("./name").text = name
-        root.find("./devices/emulator").text = self.kvm_binary
+        root.find("./devices/emulator").text = str(self.kvm_binary)
```
Comment (zloidemon) [inline virt_lightning/__init__.py]: is kvm_binary not string?

File: virt_lightning/__init__.py [hunk 3]
```diff
@@ -263,7 +276,15 @@ def root_password(self, root_password=None):
         if chpassd:
             return chpassd["list"].split(":")[1]
 
-    def ssh_key_file(self, ssh_key_file):
+    @property
+    def ssh_key(self):
+        return self._ssh_key
+
+    @ssh_key.setter
+    def ssh_key(self, value):
```
Comment (zloidemon) [inline virt_lightning/__init__.py]: In next task need validate ssh_key value for correct key. (not required for this review)

File: tests/test_hv.py (hunk 1 above)
Comment (goneri) [reply tests/test_hv.py]: It should, see: https://libvirt.org/drvtest.html

File: virt_lightning/__init__.py (hunk 2 above)
Comment (goneri) [reply virt_lightning/__init__.py]: It's a pathlib.PosixPath instance.

File: virt_lightning/__init__.py (hunk 3 above)
Comment (goneri) [reply virt_lightning/__init__.py]: Good idea! Could you open an issue to trace the idea.

**Pull request merged**